src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import './variables';
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
+    background-image: url('src/images/star-empty.png');
+    background-size: cover;
+    margin-right: $star-margin;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+  &--0 &__star:nth-child(-n + 0) {
+    background-image: url('src/images/star-filled.png');
```

Comment:
Verify that the image `src/images/star-filled.png` exists at the specified path. Incorrect paths will result in missing images.